doc/examples/simulate_dki.py

```diff
@@ -0,0 +1,79 @@
+"""
+==========================
+DKI MultiTensor Simulation
+==========================
+
+In this example we show how to simulate the diffusion kurtosis imaging (DKI)
+data of a single voxel. DKI captures information of tissue heterogeneity,
+therefore DKI simulations have to take into account different tissue
+compartments with different diffusion properties. For example, here diffusion
+heterogeneity is taken into account by modeling different compartments for the
+intra- and extra-cellular media. These simulations are performed according to:
+
+[1] R. Neto Henriques et al., "Exploring the 3D geometry of the diffusion
```

Comment:
The references should be in a section of their own at the bottom of the example.